src/main/java/bowling/step2/domain/frame/FinalFrame.java

```diff
@@ -0,0 +1,57 @@
+package bowling.step2.domain.frame;
+
+import bowling.step2.domain.status.*;
+
+import java.util.Objects;
+
+public class FinalFrame implements Frame {
+	private Status status;
+	private Status bonusStatus;
+	private boolean gameOver = false;
+
+	public FinalFrame() {
+		status = new Ready();
+	}
+
+	@Override
+	public Frame bowling(int pin) {
+		status = status.bowling(pin);
+
+		gameOver = false;
+		if (!status.isFinished()) {
+			return this;
+		}
+		if (!Objects.isNull(bonusStatus)) {
+			status = bonusStatus.bowling(pin);
+		}
+		if (status instanceof Strike || status instanceof Spare) {
```

Comment:
Status 에 isClear 메서드를 추가해보면 어떨까요? 😄 
```suggestion
		if (status.isClear()) {
```